#! /bin/sh -e
## 2_newpatch.dpatch by Roland Mas <lolando@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: muttrc-mode.el: change default location of the Mutt manual

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
diff -urNad 2.tmp/elisp/emacs-goodies-el/muttrc-mode.el 2/elisp/muttrc-mode.el
--- 2.tmp/elisp/emacs-goodies-el/muttrc-mode.el	2003-01-05 17:10:41.000000000 +0100
+++ 2/elisp/muttrc-mode.el	2003-01-05 17:27:33.000000000 +0100
@@ -89,7 +89,7 @@
   :group 'files
   :prefix "muttrc-")
 
-(defcustom muttrc-manual-path "/usr/share/doc/mutt/manual.txt"
+(defcustom muttrc-manual-path "/usr/share/doc/mutt/manual.txt.gz"
   "Path to the Mutt manual."
   :type 'string
   :group 'muttrc)
